src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -79,11 +80,12 @@ function verify(): RequestMeta {
  * @return {RequestMeta}                Request object that can be passed to fetch() API
  */
 const userCreate = (keys: UserKeys): RequestMeta => {
```

Comment:
Instead of adding the `needsRotation` to the `UserKeys` type (which makes the name less honest) and argument here, I'd just add a second boolean argument for `needsRotation`. That change will end up cascading down to the `callUserCreateApi` method which should take separate arguments for `keys` and `needsRotation` in this file as well as the code that calls that function.